pkg/provider/azure_local_services.go

```diff
@@ -47,15 +58,15 @@ type batchOperation interface {
 
 // loadBalancerBackendPoolUpdateOperation is an operation that updates the backend pool of a load balancer.
 type loadBalancerBackendPoolUpdateOperation struct {
+	serviceName      string
 	loadBalancerName string
 	backendPoolName  string
 	kind             consts.LoadBalancerBackendPoolUpdateOperation
 	nodeIPs          []string
-	result           chan batchOperationResult
 }
 
 func (op *loadBalancerBackendPoolUpdateOperation) wait() batchOperationResult {
-	return <-op.result
+	return batchOperationResult{}
```

Comment:
Is this still used somewhere? seems it should be deleted.

Reply:
Yes, it is used in route updater.